packages/ui/src/app/pages/Forum/ForumArchived.tsx

```diff
@@ -38,8 +40,23 @@ export const ForumArchived = () => {
       main={
         <>
           <RowGapBlock gap={24}>
-            <ItemCount count={forumCategories?.length}>Archived Categories</ItemCount>
-            {isLoadingCategories ? <Loading /> : forumCategories && forumCategories.length > 0 && <ForumMain />}
+            {isLoadingCategories ? (
+              <Loading />
+            ) : (
+              forumCategories &&
+              forumCategories.length > 0 && (
+                <RowGapBlock gap={10}>
+                  <TextMedium value lighter>
+                    Archived Categories
+                  </TextMedium>
```

Comment:
Also I can't find the new design for this but shouldn't we keep the number of archived categories ? and still show:
![image](https://user-images.githubusercontent.com/6571453/200009908-83c5e39d-32b4-4c99-bd7c-617718720282.png)
when there are none ?